[2022-03-14 12:48:13,540] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: main_dag.fetch_weather_data scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-14 12:48:13,631] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: main_dag.fetch_weather_data scheduled__2022-03-13T00:00:00+00:00 [queued]>
[2022-03-14 12:48:13,634] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 12:48:13,639] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-03-14 12:48:13,642] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 12:48:13,744] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2022-03-13 00:00:00+00:00
[2022-03-14 12:48:13,760] {standard_task_runner.py:52} INFO - Started process 2512 to run task
[2022-03-14 12:48:13,773] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'fetch_weather_data', 'scheduled__2022-03-13T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpac_jg4_8', '--error-file', '/tmp/tmpcz57ozrj']
[2022-03-14 12:48:13,778] {standard_task_runner.py:77} INFO - Job 5: Subtask fetch_weather_data
[2022-03-14 12:48:14,131] {logging_mixin.py:109} INFO - Running <TaskInstance: main_dag.fetch_weather_data scheduled__2022-03-13T00:00:00+00:00 [running]> on host e0810cf0708b
[2022-03-14 12:48:14,731] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=fetch_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2022-03-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-13T00:00:00+00:00
[2022-03-14 12:48:23,029] {python.py:175} INFO - Done. Returned value was: None
[2022-03-14 12:48:23,072] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=fetch_weather_data, execution_date=20220313T000000, start_date=20220314T124813, end_date=20220314T124823
[2022-03-14 12:48:23,148] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 12:48:23,267] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
